src/psychrometrics/t_mrt.js

```diff
@@ -0,0 +1,147 @@
+import { valid_range, round } from "../utilities/utilities.js";
+
+const g = 9.81;
+const c_to_k = 273.15;
+const cp_air = 1004;
+const mu = 0.0000181; // Pa s
+const k_air = 0.02662; // W/m-K
+const beta = 0.0034; // 1/K
+const nu = 0.0000148; // m2/s
+const alpha = 0.00002591; // m2/s
+const pr = (cp_air * mu) / k_air; // Prandtl constants, number
+const o = 0.0000000567;
+
+/**
+ * Converts globe temperature reading into mean radiant temperature in accordance with either the Mixed Convection developed by Teitelbaum E. et al. (2022) or the ISO 7726:1998 Standard [5]_.
+ * @see {@link t_mrt_array} for a version that supports arrays
+ * 
+ * @param {number} tg - globe temperature, [°C]
+ * @param {number} tdb - air temperature, [°C]
+ * @param {number} v - air speed, [m/s]
+ * @param {number} d - diameter of the globe, [m] default 0.15 m
+ * @param {number} emissivity - emissivity of the globe temperature sensor, default 0.95
+* @param {"Mixed Convection" | "ISO"} standard - either choose between the Mixed Convection and ISO formulations.
+        The Mixed Convection formulation has been proposed by Teitelbaum E. et al. (2022)
+        to better determine the free and forced convection coefficient used in the
+        calculation of the mean radiant temperature. They also showed that mean radiant
+        temperature measured with ping-pong ball-sized globe thermometers is not reliable
+        due to a stochastic convective bias [22]_. The Mixed Convection model has only
+        been validated for globe sensors with a diameter between 0.04 and 0.15 m.
+* @returns {number}
+*/
+
+export function t_mrt(
+  tg,
+  tdb,
+  v,
+  d = 0.15,
+  emissivity = 0.95,
+  standard = "Mixed Convection",
+) {
+  standard = standard.toLowerCase();
+  let tr = 0;
+  if (standard === "mixed convection") {
+    tr = get_tr_mixed_convection(tg, tdb, v, d, emissivity);
+    return d >= 0.04 && d <= 0.15 ? round(tr, 1) : NaN;
+  }
+
+  if (standard === "iso") {
+    return get_tr_iso(tg, tdb, v, d, emissivity);
+  }
+  throw new Error(
+    "No standard found, please choose standard from ISO and Mixed Convectioin",
+  );
+}
+
+/**
+ * Converts globe temperature reading into mean radiant temperature in accordance with either the Mixed Convection developed by Teitelbaum E. et al. (2022) or the ISO 7726:1998 Standard [5]_.
+ * Accepts array arguments. @see {@link t_mrt} for scalar arguments
```

Comment:
can you change the `@see {@link ...` to the beginning of the line. I changed also this code in t_o in main because the docs do not generate correctly if `@see` is not at the beginning

Reply:
ah I meant to the beginning of the line, [see here for an example](https://github.com/FedericoTartarini/jsthermalcomfort/blob/main/src/psychrometrics/t_o.js#L28)